scripts/install_third_party_libs.py

```diff
@@ -22,19 +22,39 @@
 import subprocess
 import sys
 
-# These libraries need to be installed before running or importing any script.
+
 TOOLS_DIR = os.path.join(os.pardir, 'oppia_tools')
-# Download and install pyyaml.
-if not os.path.exists(os.path.join(TOOLS_DIR, 'pyyaml-5.1.2')):
-    subprocess.check_call([
-        sys.executable, '-m', 'pip', 'install', 'pyyaml==5.1.2', '--target',
-        os.path.join(TOOLS_DIR, 'pyyaml-5.1.2')])
-
-# Download and install future.
-if not os.path.exists(os.path.join('third_party', 'future-0.17.1')):
-    subprocess.check_call([
-        sys.executable, '-m', 'pip', 'install', 'future==0.17.1', '--target',
-        os.path.join('third_party', 'future-0.17.1')])
+
+
+def install_prerequisites():
+    """These libraries need to be installed before running
+    or importing any script.
+    """
+    # pylint: disable=global-statement
+    # pylint: disable=unused-variable
+    global TOOLS_DIR
```

Comment:
Why is this necessary?

Reply:
I need it for path of folder where packages need to be kept.

Reply:
You aren't modifying the variable outside, so this isn't necessary.

Reply:
Done! removed the function.